### ISSUE TITLE
Why does latest version in play store have no mention here ?

### ISSUE THREAD

**Issue by bingoxo** [ProtonVPN/android-app#78]
2.9.0.82 vs 2.9.0.77

**Comment by LordRafa** [comment]
Is this repo still maintained? lastest public release is v2.11.90.17 but google play is 3.2.96.10

**Comment by AlgirdasPundzius** [comment]
Yes, the repo is still maintained, and is a mirror of our fully released versions. Fully released and stable google play releases will be mirrored to github in a week or so with F-droid metadata.

**Comment by LordRafa** [comment]
Thanks for the info, so just to clarify versions in apkmirror 3.2.87.1 and 3.2.96.9 are not considered stable?

**Comment by AlgirdasPundzius** [comment]
In general, we really try our best and consider anything what is released to public to be stable, and without any major issues.
But if certain versions don't reach full production it means they had some kind issues or we wanted to make some more adjustements before releasing something fully.

So for example 3.2.96.9 had some issues with playstore listing, which needed to be adjusted code wise, but otherwise was fine. So if you got the version from somewhere else it should be fine.

**Comment by LordRafa** [comment]
Thank you so much I just was curious about things published on apkmirror, this clarifies all
